<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EntradaMercancia</class>
 <widget class="QDialog" name="EntradaMercancia">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1043</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Entrada de Mercancía</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Procedencia :</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="comboBoxProcedencia"/>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QTableView" name="tableView">
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Cod.</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Descripción</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Cantidad</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="lineEditCod">
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="maxLength">
        <number>13</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditDesc"/>
     </item>
     <item row="1" column="2" colspan="2">
      <widget class="QLineEdit" name="lineEditUds">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="maxLength">
        <number>3</number>
       </property>
      </widget>
     </item>
     <item row="1" column="4" colspan="2">
      <widget class="QDateEdit" name="dateEditCaducidad">
       <property name="displayFormat">
        <string>yyyy-MM-dd</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>P.V.P.</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4" colspan="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Fecha caducidad</string>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="QLineEdit" name="lineEditPVP">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="maxLength">
        <number>6</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonAgregarLinea">
       <property name="text">
        <string>Agregar</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonBorrar">
       <property name="text">
        <string>Borrar linea</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonAceptar">
       <property name="text">
        <string>Aceptar entradas</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSalir">
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="text">
        <string>Salir</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="lbProductos">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboBoxProcedencia</tabstop>
  <tabstop>lineEditCod</tabstop>
  <tabstop>lineEditDesc</tabstop>
  <tabstop>lineEditUds</tabstop>
  <tabstop>dateEditCaducidad</tabstop>
  <tabstop>lineEditPVP</tabstop>
  <tabstop>pushButtonAgregarLinea</tabstop>
  <tabstop>pushButtonBorrar</tabstop>
  <tabstop>pushButtonAceptar</tabstop>
  <tabstop>tableView</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonSalir</sender>
   <signal>clicked()</signal>
   <receiver>EntradaMercancia</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>519</x>
     <y>508</y>
    </hint>
    <hint type="destinationlabel">
     <x>1041</x>
     <y>489</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
